draft-ietf-taps-interface.md

```diff
@@ -2579,19 +2579,19 @@ messageContext.add(parameter, value)
 Connection.Send(messageData, messageContext)
 ~~~
 
-The simpler form of Send, which does not take any messageContext, is equivalent to passing a default MessageContext without adding any Message Properties.
+The simpler form of `Send`, which does not take any messageContext, is equivalent to passing a default MessageContext without adding any Message Properties.
 
-If an application wants to override Message Properties for a specific message,
-it can acquire an empty MessageContext Object and add all desired Message
-Properties to that Object. It can then reuse the same messageContext Object
+If an application wants to override Message Properties for a specific Message,
+it can acquire an empty MessageContext object and add all desired Message
+Properties to that object. It can then reuse the same messageContext object
 for sending multiple Messages with the same properties.
 
 Properties can be added to a MessageContext object only before the context is used
-for sending. Once a MessageContext has been used with a Send call, further modifications
-to the MessageContext object do not have any effect on this Send call. Message Properties
+for sending. Once a MessageContext has been used with a `Send` call, further modifications
+to the MessageContext object do not have any effect on this `Send` call. Message Properties
```

Comment:
Wow... subtle: an action is called. The first occurrence, above, should therefore be action - but now, further modifications on the object don't have any effect on this particular call of the Send action.  So, here it *is* "call".